server/graphql-schema.js

```diff
@@ -754,6 +778,15 @@ const graphqlSchema = gql`
         """
         status: TestPlanReportStatus!
         """
+        Date of when the status was last updated.
+        """
+        phaseChangeUpdate: Timestamp!
+        """
+        The intended target date for the final TestPlanReport phase promotion.
+        Based on the ARIA-AT Working Mode.
+        """
+        phaseTargetDate: Timestamp
+        """
```

Comment:
> * The fact that we're calling the status both "status" and "phase" might be worth trying to avoid.

Agreed!

> * It seems conceivable that we will want to permanently save the date when each status was reached, even after the test plan has reached a recommended status (or a future deprecated status).

Makes sense.

> * I don't think the target date needs to be persisted since it's simply six months after the candidate phase starts. And I can imagine that in the real world, the date the phase is officially changed might differ from the exact six months mark, like in the case that a vendor puts in a bunch of feedback on the last day or something.

Well it isn't currently being persisted on the DB but derived from the `candidateStatusReachedAt/phaseChangeUpdate` but I do see a need where it needs to be, based on the latest designs just in terms of providing expected functionality. If it's not persisted, it would shifting the value of the `candidateStatusReachedAt` variable which doesn't seem correct to me.

> Anyway these considerations make me think of something like `candidateStatusReachedAt` and `recommendedStatusReachedAt`.

I like those!

Reply:
Yay great!